packages/auth/src/gql/getBaseAuthSchema.ts

```diff
@@ -103,7 +102,6 @@ export function getBaseAuthSchema<I extends string, S extends string> ({
           // Update system state
           const sessionToken = await context.sessionStrategy.start({
             data: {
-              listKey,
```

Comment:
note to self: this is end of any way of configuring authentication using multiple lists. (Not that I figured a good way to achieve that with `@keystone-6/core` anyways

Reply:
Well, if you're using `@keysotne-6/auth`, you can't easily use multiple lists. 
If you are writing your own `sessionStrategy`, you can do whatever you want.

Either way, I see no reason for us to break this when it's arbitrary anyway.
I'll revert the removal.

Reply:
Reflecting on this, I think I'll keep the removal.
In the unlikely event that you are wrapping `@keystone-6/auth`, you shouldn't be relying on `listKey` in the cookie and session.